### ISSUE TITLE
Feature request: Copy column

### ISSUE THREAD

**Issue by PeterP2** [trobonox/kanri#321]
Now and then it happens that you want to copy columns

When columns grow, you can think about splitting them into two columns, e.g. "recurring" into "daily/weekly recurring" and "monthly/longer recurring". The fastest way to do it then is to copy the whole column including all cards and delete the existing cards either in the old column or in the new one.

**Comment by trobonox** [comment]
Good idea, this seems related to #289 which is related to cards but very similar. It is definitely a feature I'd like to implement at some point, but needs a bit of extra thought into the technical implementation, so it will take some time and consideration.
